### ISSUE TITLE
CoreFoundation not linked with 5.4 and dev snapshots

### ISSUE THREAD

**Issue by MaxDesiatov** [swiftwasm/swift#2598]
Even when not using LTO, I'm seeing these linker errors:

```
wasm-ld: error: /Users/maxd/Library/Developer/Toolchains/swift-wasm-5.4-SNAPSHOT-2021-01-26-a.xctoolchain/usr/lib/swift_static/wasi/
libFoundation.a(PropertyListSerialization.swift.obj): undefined symbol: CFPropertyListCreateWithData
wasm-ld: error: /Users/maxd/Library/Developer/Toolchains/swift-wasm-5.4-SNAPSHOT-2021-01-26-a.xctoolchain/usr/lib/swift_static/wasi/
libFoundation.a(NSCFSet.swift.obj): undefined symbol: CFSetGetCount
wasm-ld: error: /Users/maxd/Library/Developer/Toolchains/swift-wasm-5.4-SNAPSHOT-2021-01-26-a.xctoolchain/usr/lib/swift_static/wasi/
libFoundation.a(NSCFSet.swift.obj): undefined symbol: CFSetGetValues
wasm-ld: error: /Users/maxd/Library/Developer/Toolchains/swift-wasm-5.4-SNAPSHOT-2021-01-26-a.xctoolchain/usr/lib/swift_static/wasi/
libFoundation.a(NSCFString.swift.obj): undefined symbol: CFStringGetLength
wasm-ld: error: /Users/maxd/Library/Developer/Toolchains/swift-wasm-5.4-SNAPSHOT-2021-01-26-a.xctoolchain/usr/lib/swift_static/wasi/
libFoundation.a(NSUUID.swift.obj): undefined symbol: CFHashBytes
wasm-ld: error: /Users/maxd/Library/Developer/Toolchains/swift-wasm-5.4-SNAPSHOT-2021-01-26-a.xctoolchain/usr/lib/swift_static/wasi/
libFoundation.a(UUID.swift.obj): undefined symbol: _cf_uuid_generate_random
wasm-ld: error: /Users/maxd/Library/Developer/Toolchains/swift-wasm-5.4-SNAPSHOT-2021-01-26-a.xctoolchain/usr/lib/swift_static/wasi/
libFoundation.a(UUID.swift.obj): undefined symbol: _cf_uuid_parse
wasm-ld: error: /Users/maxd/Library/Developer/Toolchains/swift-wasm-5.4-SNAPSHOT-2021-01-26-a.xctoolchain/usr/lib/swift_static/wasi/
libFoundation.a(UUID.swift.obj): undefined symbol: _cf_uuid_unparse_upper
```

**Comment by MaxDesiatov** [comment]
Current workaround is to pass all linker flags manually, as in
```
swift build --triple wasm32-unknown-wasi \
  -Xlinker -lCoreFoundation -Xlinker -licuuc \
  -Xlinker -licui18n -Xlinker -luuid -Xlinker -lBlocksRuntime
```

**Comment by MaxDesiatov** [comment]
Resolved in SwiftWasm 5.4.0
